tests/Console/Command/DescribeCommandTest.php

```diff
@@ -32,18 +34,31 @@
  */
 final class DescribeCommandTest extends TestCase
 {
+    /**
+     * @var Application
+     */
+    private $application;
+
+    protected function setUp()
+    {
+        $this->application = new Application();
```

Comment:
between tests - then the `setUp` is called again overwriting last value of the property.

to `null` - we only do it in `tests/Console/ConfigurationResolverTest.php`, it's not needed indeed, it shall be handled by phpunit after whole test file is executed